app/src/main/java/com/cornellappdev/scoop/ui/components/search/SearchFirstPage.kt

```diff
@@ -0,0 +1,79 @@
+package com.cornellappdev.scoop.ui.components.search
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.CalendarToday
+import androidx.compose.material.icons.filled.NearMe
+import androidx.compose.material.icons.filled.Place
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.scoop.R
+import com.cornellappdev.scoop.ui.components.general.CityPicker
+import com.cornellappdev.scoop.ui.components.general.DatePicker
+import com.cornellappdev.scoop.ui.theme.PlaceholderGray
+
+@Composable
+fun SearchFirstPage() {
+    val departureLocation = rememberSaveable { mutableStateOf("") }
```

Comment:
These should be in a viewmodel

Reply:
Bro give it a sec this is the first iteration T.T it's not even abt the searching flow yet